eventuals/concurrent.h

```diff
@@ -477,14 +477,16 @@ struct _Concurrent {
       static_cast<Fiber<E>*>(fiber)->k.emplace(
           Build(FiberEventual(fiber, std::move(arg))));
 
-      // NOTE: we need to "submit" using the default scheduler so that
-      // each fiber will properly be using it's own scheduler context.
-      Scheduler::Default()->Submit(
+      auto child = fiber->Clone();
+      fiber->child = std::move(child);
```

Comment:
See other comments, but this should become:
```suggestion
      // TODO(benh): differentiate the names of the fibers for
      // easier debugging!
      fiber->child.emplace("[concurrent fiber]");
```